docs/rst/user_manual/configuration.rst

```diff
@@ -313,7 +313,7 @@ A complete example of all the configurations described on this page can be found
 
```

Comment:
Could you add an index to this section, please?

Reply:
:+1: